sites/partners/src/components/listings/PaperListingDetails/sections/DetailApplicationTypes.tsx

```diff
@@ -1,13 +1,23 @@
 import React, { useContext } from "react"
 import { t, MinimalTable, StandardTableData } from "@bloom-housing/ui-components"
 import { FieldValue, Grid } from "@bloom-housing/ui-seeds"
-import { ApplicationMethodsTypeEnum } from "@bloom-housing/shared-helpers/src/types/backend-swagger"
+import {
+  ApplicationMethodsTypeEnum,
+  FeatureFlagEnum,
+} from "@bloom-housing/shared-helpers/src/types/backend-swagger"
 import { ListingContext } from "../../ListingContext"
 import { getDetailBoolean } from "./helpers"
 import SectionWithGrid from "../../../shared/SectionWithGrid"
+import { AuthContext } from "@bloom-housing/shared-helpers"
 
 const DetailApplicationTypes = () => {
   const listing = useContext(ListingContext)
+  const { doJurisdictionsHaveFeatureFlagOn } = useContext(AuthContext)
+
+  const disableCommonApplication = doJurisdictionsHaveFeatureFlagOn(
```

Comment:
Can you add a test in `/sites/partners/__tests__/pages/listings/[id]/index.test.tsx` where this returns true?